addons/hr_holidays/static/src/dashboard/time_off_card.js

```diff
@@ -21,12 +29,20 @@ export class TimeOffCard extends Component {
             approved: data.leaves_approved,
             planned: data.leaves_requested,
             left: data.virtual_remaining_leaves,
+            employeeId: this.props.employeeId,
+            holidayStatusId: this.props.holidayStatusId,
+            onClickNewAllocationRequest: this.newAllocationRequestFrom.bind(this, this.props.employeeId, this.props.holidayStatusId),
```

Comment:
I'm wondering if this doesn't work
```suggestion
            onClickNewAllocationRequest: this.newAllocationRequestFrom.bind(this)
```
and in newAllocationRequestFrom you simply use the employeeId and holidayId available in this.props.

Another option would be:
```suggestion
            onClickNewAllocationRequest: () => this.newAllocationRequestFrom(this.props.employeeId, this.props.holidayStatusId),
```

Any option is fine for me